<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="file_btn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>470</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>File Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>550</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Button</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_lable">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>561</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File Namedsafsfdsfdsfdsfsdfdsfsdfsfdsfdsfsfsdfsd</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_area">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>561</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string>image Area</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
